app/src/main/java/mozilla/lockbox/store/AccountStore.kt

```diff
@@ -224,6 +232,41 @@ open class AccountStore(
 
         this.securePreferences.remove(Constant.Key.firefoxAccount)
         this.generateNewFirefoxAccount()
+
+        webView.clearCache(true)
+        clearLogs()
+
+        dispatcher.dispatch(DataStoreAction.Reset)
+    }
+
+    private fun clearLogs() {
+        clearLogFolder(logDirectory)
+        log.info("Log pruning completed.")
+    }
+
+    private fun clearLogFolder(dir: File) {
+        if (dir.isDirectory) {
+            try {
+                val leveldbDir = dir.listFiles()
+                    .filter { file ->
+                        file.name.startsWith("Local")
+                    }[0]
+                    .listFiles()
+                    .filter { file ->
+                        file.name.startsWith("leveldb")
+                    }[0]
+                    .listFiles()
+
+                for (logs in leveldbDir) {
```

Comment:
nit: putting an `s` after something that's really a singular piece of the `for`-loop. maybe `for file in...` or `for logFile in...`.